### PR TITLE
fix(lodash): switch to normal lodash package

### PR DESCRIPTION
**Pull request by vvo**
\+ use lodash only when immediately valuable compared to default language
features

### CODE REVIEW

File: docs/examples/dev-novel/js/main.67c98e6bd1cbbf5c1f7c.js.map [hunk 1]
```diff
@@ -1 +1 @@
...erContainer\") public sliderContainer: any;\n\n  // render options\n  @Input() public pips: boolean = true;\n  @Input() public tooltips: boolean = true;\n\n  // connector options\n  @Input() public attribute: string;\n  @Input() public min?: number | string;\n  @Input() public max?: number | string;\n  @Input() public precision: number | string = 2;\n\n  public state: RangeSliderState = {\n    range: { min: 0, max: 1 },\n    refine: noop,\n    start: [0, 1]\n  };\n\n  private slider: any;\n\n  get step() {\n    // compute step from the precision value\n    const precision = parseNumberInput(this.precision) || 2;\n    return 1 / Math.pow(10, precision);\n  }\n\n  constructor(\n    @Inject(forwardRef(() => NgAisInstantSearch))\n    public instantSearchParent: any\n  ) {\n    super(\"RangeSlider\");\n  }\n\n  public ngOnInit() {\n    this.createWidget(connectRange, {\n      attributeName: this.attribute,\n      max: parseNumberInput(this.max),\n      min: parseNumberInput(this.min),\n      precision: parseNumberInput(this.precision)\n    });\n\n    super.ngOnInit();\n  }\n\n  public updateState = (state, isFirstRendering: boolean) => {\n    if (isFirstRendering) {\n      // create sli
```
Comment (Haroenv) [inline docs/examples/dev-novel/js/main.67c98e6bd1cbbf5c1f7c.js.map]: do we really need this file in this branch?

**Comment by vvo** [comment]
Situation is not so easy:
- lodash normal package + TypeScript = hell
- TypeScript now fails because some () => {} are indicating no arguments while they are called with arguments

I am no TypeScript dev for now for sure :D

**Comment by Haroenv** [comment]
`() => {}` could be changed to `(...args: any) => {}`

**Comment by samouss** [comment]
I revert the changes for `lodash` to `lodash-es` but using the `lodash-es/$NAME` not sure it will solve the issue though. We can maybe test a small Angular App on CodeSandbox that use this pattern.

**Pull request merged**